Gemfile

```diff
@@ -53,7 +53,7 @@ end
 group :test do
   # Use system testing [https://guides.rubyonrails.org/testing.html#system-testing]
   gem "capybara"
+  gem "rails-controller-testing"
```

Comment:
whats this for?

Reply:
It let's us use the `render_template` matcher in spec/views/pages/terms_of_use.html.erb_spec.rb to test that the right partial is shown. But thinking about it that is dependent on implementation detail. Perhaps it is better just to assert some content that exists in the partial?

Reply:
I've replaced this with a test that just looks at content in the partial.

Reply:
ah ok that wasn't clear in the commit message but yes also just relying on the existing test is fine